company-profiles/over.md

```diff
@@ -0,0 +1,49 @@
+# Over
+
+## Company blurb
+
+Over is in the business of brand-building. Whether personal or professional, for fun or for profit, the Over app helps build, boost, and expand brands. Our users are incredibly ambitious, nothing-is-impossible entrepreneurs, side hustlers, risk takers, and dreamers who are often time starved, resource strapped, and design challenged.
+
+We’re a Silicon Valley startup based in Cape Town, South Africa. Our team of top-notch designers, social media mavens, branding experts, ease-of-use engineers, and the most supportive customer support team on earth are on hand to help our users create everything from social posts and digital ads to logos and flyers. With professionally-designed templates, easy-to-use tools, and hand-selected, fonts, photos, and graphics, Over gives the confidence and creativity, imagination and inspiration to build any brand.
+
+Only Over is purpose-built for mobile so our users can create anywhere, at any time. No desktop required. All that is needed is the app, a phone, and your thumb to grow and thrive on social media and beyond.
+
+## Company size
+
+50-100
+
+## Remote status
+
+Work where you are MOST productive. We don't care where.
+
+We trust you to get work done and be productive wherever you chose from. There are, however, some general guides to when working remotely, particularly over a long period of time.
+
+- Over Communicate
+- Share Progress
+- Keep it Public
+- Be Available
+
+## Region
+
+Worldwide
+
+### Example region 1
+USA
+
+### Example region 2
+North America, Latin America, Caribbean
+
+### Example region 3
+Worldwide
```

Comment:
Let's remove the "Example region" stuff from this profile... @bertadevant @dougaitken can you help with this?

Reply:
Yeah I will tackle it in the morning!

Reply:
See #552.